application/healthcare-services-data-manager/hs_service.py

```diff
@@ -0,0 +1,44 @@
+import boto3
+
+TABLE_NAME = "healthcare_services"
```

Comment:
Not for this ticket maybe ? Should we add some exception handling to this code

Reply:
Might be an idea to raise a separate ticket to cover off exception handing, then someone else can have a go at that and writing the unit tests to cover it?